xbmc/guiinfo/GUIInfoLabels.h

```diff
@@ -799,30 +860,21 @@
 #define LISTITEM_ADDON_SIZE         (LISTITEM_START + 179)
 #define LISTITEM_EXPIRATION_DATE    (LISTITEM_START + 180)
 #define LISTITEM_EXPIRATION_TIME    (LISTITEM_START + 181)
+#define LISTITEM_PROPERTY           (LISTITEM_START + 182)
 
-//! @todo There are issues with the LISTITEM_PROPERTY range, breakage occurs when more than 200 properties are used in skins.
-#define LISTITEM_PROPERTY_START     (LISTITEM_START + 200)
-#define LISTITEM_PROPERTY_END       (LISTITEM_PROPERTY_START + 2500)
-#define LISTITEM_END                (LISTITEM_PROPERTY_END)
-
-#define MUSICPLAYER_PROPERTY_OFFSET       1300  // 200 id's reserved for musicplayer props.
-#define LISTITEM_ART_OFFSET               1500  // 200 id's reserved for listitem art.
-#define LISTITEM_RATING_OFFSET      1700 // 200 id's reserved for listitem ratings.
-#define LISTITEM_VOTES_OFFSET       1900 // 200 id's reserved for listitem votes.
-#define LISTITEM_RATING_AND_VOTES_OFFSET  2100 // 200 id's reserved for listitem ratingandvotes.
-#define LISTITEM_DURATION_OFFSET    2300 // 200 id's reserved for listitem duration.
+#define LISTITEM_END                (LISTITEM_START + 2500)
 
-#define CONDITIONAL_LABEL_START       LISTITEM_END + 1 // 37701
+#define CONDITIONAL_LABEL_START       (LISTITEM_END + 1) // 37501
```

Comment:
while you're at it, mind increasing CONDITIONAL_LABEL_END to 39999 ?

ref: https://forum.kodi.tv/showthread.php?tid=330522

Reply:
np, will do

Reply:
done, new commit for this.